specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -2966,6 +3607,248 @@
+        {
+          "$ref": "#/definitions/UpdateResourceDefinition"
+        }
+      ],
+      "description": "Specifies information about the gallery Script Version that you want to update."
+    },
+    "GalleryScriptVersionProperties": {
+      "type": "object",
+      "properties": {
+        "publishingProfile": {
+          "$ref": "#/definitions/GalleryScriptVersionPublishingProfile"
+        },
+        "safetyProfile": {
+          "$ref": "#/definitions/GalleryScriptVersionSafetyProfile"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/GalleryProvisioningState"
+        },
+        "replicationStatus": {
+          "readOnly": true,
+          "$ref": "#/definitions/ReplicationStatus"
+        }
+      },
+      "required": [
+        "publishingProfile"
+      ],
+      "description": "Describes the properties of a gallery script version."
+    },
+    "GalleryScriptVersionPublishingProfile": {
+      "type": "object",
+      "properties": {
+        "source": {
+          "$ref": "#/definitions/ScriptSource"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/GalleryArtifactPublishingProfileBase"
+        }
+      ],
+      "required": [
+        "source"
+      ],
+      "description": "The publishing profile of a gallery image version."
+    },
+    "GalleryScriptVersionSafetyProfile": {
+      "type": "object",
+      "properties": {},
+      "allOf": [
+        {
+          "$ref": "#/definitions/GalleryArtifactSafetyProfileBase"
+        }
+      ],
+      "description": "The safety profile of the Gallery Script Version."
+    },
+    "ScriptSource": {
+      "type": "object",
+      "properties": {
+        "scriptLink": {
+          "type": "string",
+          "description": "Required. The link of the source script, it must be a readable storage blob with SAS URI or publicly accessible URI or managed identity enabled."
+        },
+        "parameters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GalleryScriptParameter"
+          },
+          "description": "Optional. Any input parameters that needs to passed to the script and are accessed within the script for its execution."
+        }
+      },
+      "required": [
+        "scriptLink"
+      ],
+      "description": "The source script from which the Script Version is going to be created."
+    },
+    "GalleryScriptParameter": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/GenericGalleryParameter"
+        }
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Specifies the type of the Gallery Script parameter. Possible values are: String, Int, Double, Boolean, Enum",
+          "enum": [
+            "String",
+            "Int",
+            "Double",
+            "Boolean",
+            "Enum"
+          ],
+          "x-ms-enum": {
```

Comment:
>minValue [](http://example.com/codeflow?start=9&length=8)

minimumValue

Reply:
this would be a breaking change and it would break the experience. Microservices are already deployed with these names. We would like to keep them as-is